dev/NavigationView/NavigationViewItem.cpp

```diff
@@ -1070,7 +1070,8 @@ void NavigationViewItem::ProcessPointerCanceled(const winrt::PointerRoutedEventA
     // What this flag tracks is complicated because of the NavigationView sub items and the m_capturedPointers that are being tracked..
     // We do this check because PointerCaptureLost can sometimes take the place of PointerReleased events.
     // In these cases we need to test if the pointer is over the item to maintain the proper state.
-    if (IsOutOfControlBounds(args.GetCurrentPoint(*this).Position()))
+    // In the case of touch input, we want to cancel anyway since there is not pointer exited for touch events.
```

Comment:
From the [doc](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.uielement.pointerexited?view=winrt-22000#pointerexited-for-touch-input), pointer exited should have been fired for touch as well. Maybe we should just say "we want to cancel anyway".

Or are there 2 code paths that we need to handle, one for cancelling, one for existing?

Reply:
Fair point, my comment was not worded entirely correct. So if a pointer is cancelled, in my testing atleast, there won't be a pointer exited. So in the case of touch, we just assume it's gone and we should not stay in the pointer-over mode.